### ISSUE TITLE
GPU support is only for nvidia?

### ISSUE THREAD

**Issue by polarathene** [pop-os/tensorman#4]
The blog mentions CUDA and the README shows the runtime as nvidia with tensorman aliasing it to `gpu`. Is only nvidia is supported despite the generic `gpu` flag? What about AMD ROCm tensorflow support?

**Comment by mmstick** [comment]
Yes, Tensorflow only officially supports CUDA in their Docker images. There is a fork of Tensorflow with ROCm support, although it only supports version 1.14, and its not integrated here.

**Comment by polarathene** [comment]
@mmstick Are you sure? I don't have an AMD GPU myself yet to confirm, but afaik there is 2.0 support available, and the official dockerhub images are also tagged for such:

https://hub.docker.com/r/rocm/tensorflow/tags
https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/issues/362
https://www.phoronix.com/scan.php?page=news_item&px=Radeon-ROCm-2.4-Released

1.14 was only released in July, and 1.15 only just came out 4 days ago... Seems like an odd reason to not want to support it, I'm sure they'll provide a 1.15 tagged image in the near future. Isn't that kind of the purpose of this project, to support different versions/configurations easily?:

> However, configuring and managing Docker containers for Tensorflow using the docker command line is currently tedious, and managing multiple versions for different projects is even moreso. To solve this problem for our users, we have developed tensorman as a convenient tool to manage the installation and execution of Tensorflow Docker containers.

It is an [official community build](https://github.com/tensorflow/tensorflow#community-supported-builds) as listed in the tensorflow project README. Just 3 days earlier, they've sent a PR updating the README section to show [support for 1.15 and 2.x releases](https://github.com/tensorflow/tensorflow/pull/33475). AFAIK, they're working towards upstreaming as much as possible?

If this project does not want to support AMD GPUs, then perhaps it should be more clearer in the README that it's only intended to have Nvidia for GPU support? System76 offers products with AMD GPUs don't they? Isn't the use of Docker containers meant to make the difference here much easier to support in the first place?

**Comment by mmstick** [comment]
I never stated that we do not want to support AMD GPU's. That would be a weird stance. Only that the work to enable that support here has not been done yet.

**Comment by ottworks** [comment]
Has that work been done yet?
